docs/root/intro/version_history.rst

```diff
@@ -14,6 +14,7 @@ Version history
 * router: added :ref:`auto_san_validation <envoy_api_field_core.UpstreamHttpProtocolOptions.auto_san_validation>` to support overrriding SAN validation to transport socket for new upstream connections based on the downstream HTTP host/authority header.
 * router: added the ability to match a route based on whether a downstream TLS connection certificate has been
   :ref:`validated <envoy_api_field_route.RouteMatch.TlsContextMatchOptions.validated>`.
+* runtime: allows for the ability to set default values when parsing strings.
```

Comment:
We don't normally publish release notes for changes that aren't user-facing, so I think we can leave this out unless there are a lot of uses of this interface in extension projects.